net-vpn/i2p/files/i2p-0.9.32.initd

```diff
@@ -0,0 +1,19 @@
+#!/sbin/openrc-run
```

Comment:
Please don't add duplicate files. If it's no different from the previous version, please just use it.

Reply:
I'll move to a simply having `initd` and `service` then, but, to avoid modifing the previous ebuild, I'll keep the older `i2p-0.9.31.initd` and `i2p-0.9.31.service`.